core/client/src/main/java/alluxio/client/block/options/LockBlockOptions.java

```diff
@@ -68,6 +69,13 @@ public int getMaxUfsReadConcurrency() {
   }
 
   /**
+   * @return the id of the ufs which the mount of this file is mapped to
+   */
+  public long getUfsId() {
+    return mUfsId;
+  }
```

Comment:
are you asking why we include ufsId (renamed to mountId) in the lock block option? the reason of doing this is because this option is created in `createUfsBlockInStream` and passed to worker through `blockWorkerClient.lockUfsBlock(blockId, lockBlockOptions)`, so worker knows the ufs path as well as the mount id.